plugins/processors/t128_filter/t128_filter.go

```diff
@@ -66,6 +86,46 @@ func (m exactMatcher) Matches(point telegraf.Metric) bool {
 	return false
 }
 
+type regexMatcher struct {
+	tag         string
+	expressions []*regexp.Regexp
+}
+
+func (m regexMatcher) Matches(point telegraf.Metric) bool {
+	value, ok := point.GetTag(m.tag)
```

Comment:
Is there a way to say a tag may be one of a few values or missing? Just curious if that's a valid use case we should think through.

Reply:
Yea, I remember you mentioning that before. I'll create a jira to add an option for that